agent/api/statechange.go

```diff
@@ -223,29 +224,43 @@ func NewAttachmentStateChangeEvent(eniAttachment *apieni.ENIAttachment) Attachme
 	}
 }
 
+func (c *ContainerStateChange) ToFields() logger.Fields {
```

Comment:
this function is somewhat moot since we don't send container state changes anymore, but added it just for consistency with TaskStateChange code